lib/tasks/manageiq/ui_tasks.rake

```diff
@@ -126,22 +148,16 @@ unless Rake::Task.task_defined?("webpacker")
   load 'tasks/webpacker/check_webpack_binstubs.rake' # needed by verify_install
 end
 
-# This is normally handled by the lib/vmdb/inflectors.rb of manageiq, but might
-# not be loaded if this is called from this repo.  Doesn't hurt (much) to have
-# this here, and shouldn't create duplicates (won't be in place in the booted
-# application).
-ActiveSupport::Inflector.inflections do |inflect|
-  inflect.acronym "ManageIQ"
-end
-
 def asset_engines
-  all_engines = Rails::Engine.subclasses.each_with_object({}) do |engine, acc|
-    acc[engine] = engine.root.realpath.to_s
-  end
+  @asset_engines ||= begin do
```

Comment:
looks like `begin do` is not a thing ;)

Reply:
@himdel heh, looks like #smugNickisSmug got cocky a little to early.  Didn't test with that change... go figure it is the one that breaks (facepalm).

Will push a chance in a second.